synapse/lib/modules.py

```diff
@@ -53,7 +53,7 @@ def load(name):
     '''
     smod = synmods.get(name)
     if smod == None:
-        logging.info('loading syn mod: %s' % (name,))
+        logger.info('loading syn mod: %s' % (name,))
```

Comment:
leave the formatting to the `logging` module, rather than eagerly doing it here. it can be more performant when the logger (or its level) is disabled.

recommend:
```
logger.info('loading syn mod: %s', name)
```